yw_website/yw_website/apps/website/templates/account/base.html

```diff
@@ -5,7 +5,7 @@
     {% load static %}
     {% get_static_prefix as STATIC_PREFIX %}
     <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">
-    <link rel="stylesheet" type="text/css" href="{{ STATIC_PREFIX }}design/detailed_design.css">
+    <link rel="stylesheet" type="text/css" href="{{ STATIC_PREFIX }}design/common.css">
```

Comment:
oh good catch

Reply:
> @clundin55 thanks for doing that i know its annoying

No biggie I wrongly assumed it was already in there